ofnetAgent.go

```diff
@@ -127,16 +128,17 @@ const (
 // Create a new Ofnet agent and initialize it
 /*  routerInfo[0] -> Uplink nexthop interface
  */
-func NewOfnetAgent(bridgeName string, dpName string, localIp net.IP, rpcPort uint16,
```

Comment:
are we expecting the knob to diruptive ? Meaning once you set the arp mode we can change it ? We could make it configurable ? which would mean not set the arp mode during ofnet agent creation.